validphys2/src/validphys/scripts/vp_comparefits.py

```diff
@@ -184,6 +183,11 @@ def get_commandline_arguments(self, cmdline=None):
             args['config_yml'] = comparefittemplates.template_pol_path
         else:
             args['config_yml'] = comparefittemplates.template_path
+
+        #if args.get('current_fit') and args.get('reference_fit'):
+        #    if self.check_identical_theory_cuts_covmat(args):
+        log.info("Using excluded comparecard: identical theory cuts/covmat detected.")
+        args['config_yml'] = comparefittemplates.template_with_excluded_path
         return args
 
     def complete_mapping(self):
```

Comment:
This was almost done!

You just need to put the conditional here (well, at the end of this function)

```
are_the_same = self.check_identical_theory_cuts_covmat()
if are_the_same:
    log.info("Using excluded comparecard: identical theory cuts/covmat detected.")
    autosettings["template"] = "report_with_excluded.md"
```

These `autosettings` of `complete_mapping` is what is used to complete the template afterwards.
In this particular case (in which you are just changing the `template` but would be using the exact same runcard for everything else) I would not add a new `comparecard_excluded.yaml` but just the `report_with_excluded.md`